src/pages-and-resources/discussions/app-config-form/apps/shared/discussion-topics/DiscussionTopics.jsx

```diff
@@ -4,7 +4,7 @@ import { Button } from '@openedx/paragon';
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { FieldArray, useFormikContext } from 'formik';
 import { v4 as uuid } from 'uuid';
-import _ from 'lodash';
+import { remove as removeElements, uniq, uniqBy } from 'lodash/remove';
```

Comment:
```suggestion
import { remove as removeElements, uniq, uniqBy } from 'lodash';
```
This was causing failures at runtime / test time (you can't import `uniq` from the `remove` file). Not caught sooner because this file isn't using TypeScript.